pkg/option/config.go

```diff
@@ -2157,6 +2157,12 @@ func (c *DaemonConfig) TunnelingEnabled() bool {
 	return c.Tunnel != TunnelDisabled
 }
 
+// IptablesMasqueradingEnabled returns true if iptables-based
+// masquerading is enabled.
+func (c *DaemonConfig) IptablesMasqueradingEnabled() bool {
+	return !c.EnableBPFMasquerade && (c.EnableIPv4Masquerade || c.EnableIPv6Masquerade)
```

Comment:
nit: as `||` has higher precedence than `&&`, the parentheses are not needed.

Reply:
Are you sure? `fmt.Println(false && false || true)` returns `true`.